cmd/lotus-storage-miner/retrieval-deals.go

```diff
@@ -154,3 +159,112 @@ var retrievalDealsListCmd = &cli.Command{
 		return w.Flush()
 	},
 }
+
+var retrievalSetAskCmd = &cli.Command{
+	Name:  "set-ask",
+	Usage: "Configure the provider's retrieval ask",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "price",
+			Usage: "Set the price of the ask for retrievals (per byte)",
+		},
+		&cli.StringFlag{
+			Name:  "unseal-price",
+			Usage: "Set the price to unseal",
```

Comment:
We should note the unit (e.g `FIL/GiB`)

Reply:
I believe this should be on the retrieval price, not on the unseal price.